cf-Invalidation-service/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "invalidate CF",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "dependencies": {
+    "aws-sdk": "^2.649.0"
+  },
+  "devDependencies": {},
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "Tim Jodoin",
+  "license": "ISC"
```

Comment:
Should be MIT.

Reply:
Not apache?

Reply:
Covid Alert Server is licensed under Apache as it is a fork of the [CovidShield Server Project](https://github.com/CovidShield/server)

CDS defaults to the MIT License for all of it's projects, we might deviate from this for specific reasons. 

You can always check through the Github repo either by looking at [the LICENSE file for the project](https://github.com/cds-snc/covid-alert-server-qa-tools/blob/master/LICENSE) or through the projects home page as Github will parse the LICENSE file and display the LICENSE used. 

![image](https://user-images.githubusercontent.com/87738/105047011-d0efa700-5a37-11eb-839f-a40b6acebafd.png)